android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -158,6 +158,19 @@ int getThreadCount(int threads) {
   return (threads == 0) ? android_getCpuCount() : threads;
 }
 
+// Checks if there is a pending JNI exception that will be thrown when the
+// control returns to the java layer. If there is none, it will return false. If
+// there is one, then it will clear the pending exception and return true.
+// Whenever this function returns true, the caller should treat it as a fatal
+// error and return with a failure status at the earliest instance possible.
```

Comment:
Nit instance => instant ?

Or change "at the earliest instance possible" to "as early as possible".

Reply:
Fixed in https://github.com/AOMediaCodec/libavif/pull/1408.